soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -484,121 +484,121 @@ void InitTrickNames() {
+     Text{"Din's Pearl", "Perle de Din", "Orbe de Din"},
+     Text{"Crimsonine", "Alzanine", "Bermellina"}};
   trickNameTable[0xCD] = {
      Text{"Pendant of Wisdom", "Pendentif de la sagesse", "Colgante de la sabiduría"},
-     Text{"Nayru's Sapphire", "Saphir de Nayru", "Zafiro de Nayru"},
-     Text{"Zora's Aquamarine", "Aquamarine Zora", "Lapislázuli de los Zora"}};
+     Text{"Nayru's Pearl", "Perle de Nayru", "Orbe de Nayru"},
+     Text{"Azurine", "Aquanine", "Azurina"}};
   trickNameTable[0xCE] = {
-     Text{"Wind Medallion", "Médaillon du vent", "Medallón del Viento"},
+     Text{"Wind Element", "Elément Vent", "Elemento de aire"},
      Text{"Saria's Medallion", "Médaillon de Saria", "Medallón de Saria"},
-     Text{"Medallion of Forest", "Médaillon du Temple de la Forêt", "Medalla del Bosque"}};
+     Text{"Sign of Air", "Glyphe de l'air", "Glifo de aire"}};
   trickNameTable[0xCF] = {
-     Text{"Bombos Medallion", "Médaillon des flammes", "Medallón del Temblor"},
+     Text{"Fire Element", "Elément Feu", "Elemento de fuego"},
      Text{"Darunia's Medallion", "Médaillon de Darunia", "Medallón de Darunia"},
-     Text{"Medallion of Fire", "Médaillon du Temple du Feu", "Medalla del Fuego"}};
+     Text{"Sign of Fire", "Glyphe de feu", "Glifo de fuego"}};
   trickNameTable[0xD0] = {
-     Text{"Ice Medallion", "Médaillon de glace", "Medallón Helado"},
+     Text{"Water Element", "Elément Eau", "Elemento de agua"},
      Text{"Ruto's Medallion", "Médaillon de Ruto", "Medallón de Ruto"},
-     Text{"Medallion of Water", "Médaillon du Temple de l'Eau", "Medalla del Agua"}};
+     Text{"Sign of Water", "Glyphe de l'eau", "Glifo de agua"}};
   trickNameTable[0xD1] = {
-     Text{"Quake Medallion", "Médaillon des secousses", "Medallón Llamarada"},
+     Text{"Earth Element", "Elément Terre", "Elemento de tierra"},
      Text{"Nabooru's Medallion", "Médaillon de Nabooru", "Medallón de Nabooru"},
-     Text{"Medallion of Spirit", "Médaillon du Temple de l'Esprit", "Medalla del Espíritu"}};
+     Text{"Sign of Earth", "Glyphe de la Terre", "Glifo de la tierra"}};
   trickNameTable[0xD2] = {
-     Text{"Travel Medallion", "Amulette de téléportation", "Medallón Maligno"},
+     Text{"Fused Shadow", "Cristal d'ombre", "Sombra Fundida"},
      Text{"Impa's Medallion", "Médaillon d'Impa", "Medallón de Impa"},
-     Text{"Medallion of Shadow", "Médaillon du Temple de l'Ombre", "Medalla de la Sombra"}};
+     Text{"Sign of Illusion", "Glyphe de l'illusion", "Glifo de ilusión"}};
   trickNameTable[0xD3] = {
-     Text{"Ether Medallion", "Médaillon d'éther", "Medallón de Tesoro"},
+     Text{"Compass of Light", "Boussole de lumière", "Brújula de Luz"},
      Text{"Rauru's Medallion", "Médaillon de Rauru", "Medallón de Rauru"},
-     Text{"Medallion of Light", "Médaillon du temple de lumière", "Medalla de la Luz"}};
+     Text{"Sign of Destiny", "Glyphe du destin", "Glifo del destino"}};
```

Comment:
Same with all of these, old ones were better references

Reply:
As it's not obvious, I'll note here that the Signs are a reference to Zelda's adventure where they are the main quest item.